data/blog/bitcoin-infrastructure-this-decade.mdx

```diff
@@ -0,0 +1,39 @@
+
+Given these goals, what does a bitcoin infrastructure look like a decade from now?
+
+- **Mining**, particularly transaction inclusion decisions, and block construction are more distributed, returning true value to the mempool as a necessary gossip protocol for the network.
+- **Mempool:** Improvements to the mempool (clusters, package relay, ephemeral anchors) help bitcoin scale through more robust and miner-compatible open infrastructure, providing every bitcoin node with reasonable access to competitive information and rate-setting in the global fee market.
+- **Layer 2:** A robust and interoperable set of options for "second layer" transaction ledgers, which offer bitcoiners the opportunity to stack and transact at different scales and timeline tradeoffs.
+- **Self-custody:** Accessible and understandable self-custodial tools and education, particularly opportunities for increasing the availability of self-custodianship of bitcoin in layer-2s.
+- **Privacy:** Ongoing research into new protocols that improve bitcoin privacy at the base layer, such as increasing the surface area for multi-party input transactions (Splicing, Payjoin) and network protocols, which increase anonymity and reduce traceability across the public ledger.
+- **Cryptography:** Advances in cutting-edge cryptography (ZKPs, etc.) build new options for trustless light clients, increasing the accessibility of self-sovereign bitcoin to low-end consumer hardware and in regions of spotty internet.
+- **Education** projects, particularly ones that focus on furthering understanding of the global ledger and bitcoin protocols, building a stronger technical understanding, and empowering individuals as self-sufficient agents and bitcoin holders.
+- **Fuzzing** tools and cryptographic research have advanced our confidence in the various implementations of the bitcoin protocol at every level of the stack.
+
+Contributing to bitcoin-core is a well-regarded and compensated career choice, with a thriving ecosystem of both anonymous and well-known contributors with a variety of funding sources who collaborate to make bitcoin-core more scalable, robust, and modular.
+
+## What will the next decade of bitcoin infrastructure building bring?
+
+At OpenSats, we're committed to funding existing and new builders who can help extend and re-enforce the bitcoin tools and ecosystem. We're absolutely thrilled to be able to contribute to supporting the next decade of engineers, protocol designers, and educators who are committed to maintaining and building bitcoin: a robust consensus network that's the most decentralized, accessible, and privacy-preserving as we can humanly accomplish.
```

Comment:
Use capitalized Bitcoin instead of bitcoin in line 32 in the two places where the word is used.